internal/core/data/router.go

```diff
@@ -109,19 +111,35 @@ func LoadRestRoutes(dic *di.Container) *mux.Router {
 	e.HandleFunc(
 		"/"+SCRUB,
 		func(w http.ResponseWriter, r *http.Request) {
-			scrubHandler(w, r, bootstrapContainer.LoggingClientFrom(dic.Get), container.DBClientFrom(dic.Get))
+			scrubHandler(
+				w,
+				r,
+				bootstrapContainer.LoggingClientFrom(dic.Get), container.DBClientFrom(dic.Get),
+				dataContainer.HttpErrorHandlerFrom(dic.Get))
+
 		}).Methods(http.MethodDelete)
 
 	e.HandleFunc(
 		"/"+SCRUBALL,
 		func(w http.ResponseWriter, r *http.Request) {
-			scrubAllHandler(w, r, bootstrapContainer.LoggingClientFrom(dic.Get), container.DBClientFrom(dic.Get))
+			scrubAllHandler(
+				w,
+				r,
+				bootstrapContainer.LoggingClientFrom(dic.Get), container.DBClientFrom(dic.Get),
+				dataContainer.HttpErrorHandlerFrom(dic.Get))
+
```

Comment:
Empty line should be removed (https://github.com/edgexfoundry/edgex-go/pull/2241)

Commented here; occurs elsewhere.